components/image-builder-mk3/workspace-image-layer/gitpod-layer/prepend.dockerfile

```diff
@@ -1,7 +0,0 @@
-# Copyright (c) 2020 Gitpod GmbH. All rights reserved.
-# Licensed under the GNU Affero General Public License (AGPL).
-# See License-AGPL.txt in the project root for license information.
-
-
-USER ROOT
-COPY ./gitpod-cli /usr/bin/gp
```

Comment:
> This PR removes the need for the Gitpod layer, by moving the user and group creation to supervisor.

What about the `gp` CLI? Surely we want to continue installing it as well. (Should at least test that it's still there.)